numpy/ma/core.py

```diff
@@ -2755,6 +2755,54 @@ class MaskedArray(ndarray):
         in any order (either C-, Fortran-contiguous, or even discontiguous),
         unless a copy is required, in which case it will be C-contiguous.
 
+    Examples
+    --------
+
+    The ``mask`` can be initialized with an array of boolean values
+    with the same shape as ``data``.
+    >>> data = np.arange(6).reshape((2, 3))
+    >>> np.ma.MaskedArray(data, mask=[[False, False, False],
+    ...                               [False, False, False]])
+    masked_array(
+      data=[[0, 1, 2],
+            [3, 4, 5]],
+      mask=[[False, False, False],
+            [False, False, False]],
+      fill_value=999999)
+    >>> np.ma.MaskedArray(data, mask=[[False, True, False],
+    ...                               [False, False, True]])
+    masked_array(
+      data=[[0, --, 2],
+            [3, 4, --]],
+      mask=[[False,  True, False],
+            [False, False,  True]],
+      fill_value=999999)
+
+    Alternatively, the ``mask`` can be initialized to homogeneous boolean
+    array with the same shape as ``data`` by passing in scalar boolean value:
+    >>> np.ma.MaskedArray(data, mask=False)
+    masked_array(
+      data=[[0, 1, 2],
+            [3, 4, 5]],
+      mask=[[False, False, False],
+            [False, False, False]],
+      fill_value=999999)
+
+    >>> np.ma.MaskedArray(data, mask=True)
+    masked_array(
+      data=[[--, --, --],
+            [--, --, --]],
+      mask=[[ True,  True,  True],
+            [ True,  True,  True]],
+      fill_value=999999,
+      dtype=int64)
+
+    Recommended practise for initializing ``mask`` with scalar boolean value
+    is to use ``True``/``False`` rather than ``np.True_``/``np.False_``.
+    The reason is:
```

Comment:
as demonstrated in PR [#13980](https://github.com/numpy/numpy/pull/13980/files),you should add the explanation ``mask=np.False_`` will return a scalar boolean instead of keeping the shape of `data` as well.

Reply:
Does it really concern us?  `mask=np.ma.nomask` returns scalar boolean and doesn't keep the shape of `data` and I think it's ok. The core problem is that `mask=np.False_` is actually `mask=np.ma.nomask` and behaves like it.